lazyflow/slot.py

```diff
@@ -108,6 +105,7 @@ def __init__(
         allow_mask=False,
         subindex=(),
         top_level_slot=None,
+        write_logs=False,
```

Comment:
This is kinda neat, to enable per slot debug messages which would otherwise be overkill via a config. Seeing this I wonder if this would not also be nice to have selective per instance logging for only some operators.

Reply:
I actually thought it was a bit ugly, adding a parameter and what feels like too much logic purely for logging. Plus, it's effectively logging config outside of the logging_config. But in a pixel classification example, this prevents bloating the already existing 469 loggers up to 901 loggers, so I figured it might be worth it...

Reply:
Yeah, I had the same thoughts first, but then I tried it out and found it convenient :)